shared/templates/accounts_password/oval.template

```diff
@@ -24,5 +28,18 @@
     <ind:subexpression datatype="int" operation="{{{ OPERATION }}}" var_ref="var_password_pam_{{{ VARIABLE }}}" />
   </ind:textfilecontent54_state>
 
+  {{% if CHECK_ZERO_BOUNDARY %}}
+  <ind:textfilecontent54_test check="all"
+  comment="check the configuration of /etc/security/pwquality.conf"
+  id="test_password_pam_pwquality_{{{ VARIABLE }}}_zero_boundary" version="1">
+    <ind:object object_ref="obj_password_pam_pwquality_{{{ VARIABLE }}}" />
+    <ind:state state_ref="state_password_pam_{{{ VARIABLE }}}_zero_boundary" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_state id="state_password_pam_{{{ VARIABLE }}}_zero_boundary" version="1">
```

Comment:
You can also add the state `state_password_pam_{{{ VARIABLE }}}_zero_boundary` to the test `test_password_pam_pwquality_{{{ VARIABLE }}}`. Then the object will have to conform to both states.

If you'd like to be more explicit `state_operator="AND"` could be added to the test too.
Example of `state_operator` use: https://github.com/ComplianceAsCode/content/blob/2b022d2af8c5c3f18b2bc123df3638b90248a532/linux_os/guide/system/accounts/accounts-restrictions/root_logins/no_shelllogin_for_systemaccounts/oval/shared.xml#L155